Tests/Marketplace/marketplace_services.py

```diff
@@ -2411,6 +2425,23 @@ def get_packs_statistics_dataframe(bq_client):
     return packs_statistic_table
 
 
+def get_trending_packs(bq_client) -> list:
+    """
+    Updates the landing page sections data with Trending packs.
+    Trending packs: top 20 downloaded packs in the last 14 days.
+    Args:
+        bq_client (google.cloud.bigquery.client.Client): The bigquery client with proper permissions to execute the query.
+    Returns:
+        A list with 20 pack names that has the highest download rate.
+    """
+    query = f"SELECT pack_name FROM `{GCPConfig.DOWNLOADS_TABLE}` ORDER BY num_count DESC LIMIT 20"
```

Comment:
I think you are querying the wrong table. We want to query: `oproxy-dev.shared_views.top_packs_14_days`